phpBB/phpbb/storage/stream_interface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\storage;
+
+interface stream_interface
+{
+	/**
+	 * Reads a file as a stream.
+	 *
+	 * @param string	$path	File to read
+	 *
+	 * @throws \phpbb\storage\exception\exception		When cannot open file
+	 * @throws \phpbb\storage\exception\not_implemented	When the adapter doesnt implement the method
```

Comment:
removes this line. If the adapter does not support streams then it does not add the interface